book/src/theme-gallery/index.md

```diff
@@ -106,3 +107,28 @@ It features a dominant partition of space into a bright and a dark side.
 ```
 ![bipartite theme screenshot](./bipartite.png)
 
+## University
+_[go to top](#theme-gallery)_
+
+This theme is a simple and configurable theme.
+
+### Options
+- `university-name`: The University name
+- `color-a`: A color of your choice
+- `color-b`: A color of your choice
+- `color-c`: A color of your choice
+- `logo`: A file path for a logo image to appear on the title slide only
+
+### Variants
+- `"wake up"`: no decoration, colored background, enlarged text
+  Available options:
+    - `color`: The background color
+- `"split v"`: split a slide vertically in `columns` parts
+  Available options:
+    - `columns`: integer, amount of columns
+- `"split h"`: split a slide horizontally in `rows` parts
+  Available options:
+    - `rows`: integer, amount of rows
+- `"four split"`: divide the slide in four area
+
+![university theme screenshot](./university.png)
```

Comment:
Before the screenshot, there should be the code generating it, see the other themes already present.

Reply:
fixed.